README.md

```diff
@@ -199,6 +199,18 @@ Setting this option will not verify if the http method allows for a body.
 Override the `'Content-Type'` header of the forwarded request, if we are
 already overriding the [`body`][body].
 
+#### `logProxyError(log, err)`
```

Comment:
I would use the `request` object instead the `req.log` only as first parameter.

Since the feature aims to customize the logger, users may have some decorators they want to print out or some special headers

Do you agree?

Reply:
Happy to make that change. And I agree to that use case. I will wait for additional feedback, so I can get one last commit before hopefully merging.